src/open_inwoner/cms/cases/tests/test_contactform.py

```diff
@@ -296,6 +296,15 @@ def _setUpExtraMocks(self, m):
             antwoord="",
             text="hey!\n\nwaddup?",
         )
+        self.objectcontactmoment = generate_oas_component_cached(
+            "cmc",
+            "schemas/ObjectContactMoment",
+            uuid="77880671-b88a-44ed-ba24-dc2ae688c2ec",
+            url=f"{CONTACTMOMENTEN_ROOT}objectcontactmomenten/77880671-b88a-44ed-ba24-dc2ae688c2ec",
+            object=self.zaak["url"],
+            object_type="zaak",
+            contactmoment=self.contactmoment["url"],
+        )
```

Comment:
This functionality is not actually tested in this file, right? If that is the case, I think I'd be in favor of mocking (like you did for `test_multiple_file_upload` as well)